x/inflation/types/inflation_calculation_test.go

```diff
@@ -27,7 +27,7 @@ var ExpectedYearlyInflation = []sdk.Dec{
 // = (60% of the total supply) - (Community supply at start)
 // = (60% of 1.5 billion) - (Community supply at start)
 // = 810.6 million NIBI
-var ExpectedTotalInflation = sdk.NewDec(810_600_000_000_000)
+var ExpectedTotalInflation = sdk.NewDec(807_735_002_000_000)
```

Comment:
Shouldn't this be 807.735M flat?

Reply:
there was a rounding issue which led to 2 additional nibi. Updating that!